app.py

```diff
@@ -102,25 +108,29 @@ def process_import_function(event, context):
             logger.error(f"Error downloading file: {e}")
             all_files.remove(filedir)
             continue
-
     # Process imports for all files per jurisdiction in a batch
     for abbreviation, juris in unique_jurisdictions.items():
-        logger.info(f"importing {juris['id']}...")
-
+        file_path = juris["keys"]
+        jur_id = juris["id"]
+        logger.info(f"importing {jur_id}...")
         try:
-            do_import(juris["id"], f"{datadir}{abbreviation}")
+            do_import(jur_id, f"{datadir}{abbreviation}")
             stats.send_last_run(
```

Comment:
Made a PR to update this metric collection for grafana [here](https://github.com/openstates/openstates-realtime/pull/17) - the import here will happen but the metric collection (and everything after in the try block) will fail because, the method used for metric collection has been updated in os-core and not in  this repo.